mvcc/backend/backend_test.go

```diff
@@ -300,6 +300,8 @@ func TestBackendWritebackForEach(t *testing.T) {
 	}
 }
 
+// TODO: add a unit test for concurrentReadTx
```

Comment:
can you add a test?

Reply:
> @jingyih look good to me overall.
> 
> i think we should add a random test to make sure the concurrent read works correctly against writes. Basically, we randomly create read and issue writes, and make sure the read can read the latest write value.

Thanks for the review! I agree more tests are needed.